index.js

```diff
@@ -1,7 +1,9 @@
 /*! bittorrent-lsd. MIT License. WebTorrent LLC <https://webtorrent.io/opensource> */
-const dgram = require('dgram')
-const EventEmitter = require('events').EventEmitter
-const debug = require('debug')('bittorrent-lsd')
+import dgram from 'dgram'
+import { EventEmitter } from 'events'
+import debugFactory from 'debug'
+
+const debug = debugFactory('bittorrent-lsd')
```

Comment:
Thoughts on

```suggestion
import Debug from 'debug'

const debug = Debug('bittorrent-lsd')
```

Reply:
https://github.com/webtorrent/webtorrent/blob/master/lib/torrent.js#L41

no, I wanted it consistent with other repos

Reply:
I'd rather switch there too and not bring the design patterns concept words into code.